core/src/actions/schedules.ts

```diff
@@ -116,6 +116,7 @@ export class ScheduleEdit extends AuthenticatedAction {
       name: { required: false },
       sourceId: { required: false },
       recurring: { required: false },
+      confirmProfiles: { required: false },
```

Comment:
Probably should be added as an option to `schedule:create` as well.  

Please also test the Actions with these new params.